upgrade-ls

```diff
@@ -91,8 +91,31 @@ if [[ "$UBUNTU_VERSION" == "20.04" ]]; then
       mv -v "/usr/lib/node_modules/lamassu-server" "/usr/lib/node_modules/${BKP_NAME}" >> ${LOG_FILE} 2>&1
   fi
 
+  if ! [ -f verify ]
+  then
+    decho "Downloading verifier..."
+    curl -OL https://raw.githubusercontent.com/lamassu/lamassu-install/grand-gilgamesh/verify
+    chmod +x verify
+  fi
+
+  decho "Downloading update package..."
+  curl -o /tmp/lamassu-server-v8.1.0.tar.gz -sSL https://github.com/lamassu/lamassu-server/releases/download/v8.1.0/lamassu-server-v8.1.0.tar.gz
+
+  curl -o /tmp/sig1.asc -sSL https://github.com/lamassu/lamassu-server/releases/download/v8.1.0/sig1.asc
+  curl -o /tmp/sig2.asc -sSL https://github.com/lamassu/lamassu-server/releases/download/v8.1.0/sig2.asc
+
+  decho "Verifying update package..."
+  ./verify /tmp/lamassu-server-v8.1.0.tar.gz /tmp/sig1.asc /tmp/sig2.asc
+
+  if [ $? -eq 0 ] then
+    decho "Package sucessfully verified"
+  else
+    decho "Package verification failed. Aborting update."
+    exit 5
+  fi
```

Comment:
We should be deleting `/tmp/lamassu-server-v8.1.0.tar.gz`, `/tmp/sig1.asc` and `/tmp/sig2.asc` once the verification process is finished